modules/microshift-get-rpm-release-info.adoc

```diff
@@ -0,0 +1,82 @@
+
+.Prerequisites
+
+* You are using {op-system-full} or an operating system with an RPM package manager.
+* Your {microshift-short} repositories are enabled.
+
+.Procedure
+
+. Optional: List which versions of the {microshift-short} RPM are available for download by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf repoquery microshift-release-info-0:4.16.* # <1>
+----
+<1> Replace the example value `4.16.*` with the major and minor release numbers you are interested in.
++
+.Example output
+[source,terminal]
+----
+Updating Subscription Management repositories.
+microshift-release-info-0:4.16.0-202406260523.p0.gc5a37df.assembly.4.16.0.el9.noarch
+microshift-release-info-0:4.16.1-202406281132.p0.g8babeb9.assembly.4.16.1.el9.noarch
+microshift-release-info-0:4.16.10-202408291007.p0.g6e4ee4d.assembly.4.16.10.el9.noarch
+microshift-release-info-0:4.16.2-202407040825.p0.g2e0407e.assembly.4.16.2.el9.noarch
+microshift-release-info-0:4.16.3-202407111123.p0.ge4206d3.assembly.4.16.3.el9.noarch
+microshift-release-info-0:4.16.4-202407191908.p0.g057a9af.assembly.4.16.4.el9.noarch
+microshift-release-info-0:4.16.5-202407250951.p0.g0afcb57.assembly.4.16.5.el9.noarch
+microshift-release-info-0:4.16.6-202408010822.p0.gc4ded66.assembly.4.16.6.el9.noarch
+microshift-release-info-0:4.16.7-202408081107.p0.g0597bb8.assembly.4.16.7.el9.noarch
+microshift-release-info-0:4.16.8-202408150851.p0.gc8a3bb1.assembly.4.16.8.el9.noarch
+microshift-release-info-0:4.16.9-202408220842.p0.gefa92a2.assembly.4.16.9.el9.noarch
+----
+
+. Download the RPM package you want by running the following command:
++
+--
+[source,terminal]
+----
+$ sudo dnf download microshift-release-info-_<release_version>_ # <1>
+----
+<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, for example, `4.16.0`.
+--
++
+--
+.Example output
+[source,terminal]
+----
+microshift-release-info-4.16.0.*.el9.noarch.rpm # <1>
+----
+<1> The `*` represents the date and commit ID. Your output should contain both, for example, `-202311101230.p0.g7dc6a00.assembly.4.16.0`.
+--
+
+. Unpack the RPM package without installing it by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio _<my_microshift_release_info>_ | cpio -idmv # <1>
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+----
+<1> Replace `_<my_microshift_release_info>_` with the name of the RPM package from the previous step. For example, `microshift-release-info-4.16.10-202408291007.p0.g6e4ee4d.assembly.4.16.10.el9.noarch.rpm`.
++
+.Example output
+[source,terminal]
+----
+./usr/share/microshift
+./usr/share/microshift/blueprint
+./usr/share/microshift/blueprint/blueprint-aarch64.toml
+./usr/share/microshift/blueprint/blueprint-x86_64.toml
+./usr/share/microshift/release
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
```

Comment:
maybe we can provide example of `cat` the release file to show which images will be pulled?

Reply:
Hey @vwalek so that requires jq? We have a prohibition in the docs style guidelines against it, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#commands-with-jq. Full disclosure--we do use it elsewhere in MicroShift, but I'm not sure it is worth the battle to add it here. WDYT?

Reply:
Hey, I would use the `jq` as it is easiest way to get the content of the file. The docs say `To see the images included with a MicroShift release, you can list the contents of the microshift-release-info RPM by downloading and unpacking the RPM.` -> but there is no example on how to check it.

We can put just `cat ./usr/share/microshift/release/release-x86_64.json` to list the content.

```
{
  "release": {
    "base": "4.15.29"
  },
  "images": {
    "cli": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:8442242b4ffccbcfea5838515e571a659722bc1eba5478c05b19e89da64c2eb2",
    "coredns": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a459dc49c781913cfad0931f25b0c86914a01c8c18c8534e57b19dadcee6a1a6",
    "haproxy-router": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fec5533c975243477bd2157a989f1cae32d57657f56fa4e5849edefd2d2d3fc1",
    "kube-rbac-proxy": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9d82f824c6e96602bc4b423d11f31998bb0789b8a7d9ab820997270c9f93b43c",
[...]
```